packages/syft/src/syft/core/node/new/task/oblv_service.py

```diff
@@ -0,0 +1,438 @@
+# stdlib
+from base64 import encodebytes
+import os
+import random
+import subprocess  # nosec
+from typing import Any
+from typing import Callable
+from typing import Dict
+from typing import List
+from typing import Optional
+from typing import Tuple
+from typing import cast
+
+# third party
+from oblv import OblvClient
+import requests
+from result import Err
+from result import Ok
+from result import Result
+
+# relative
+from .....core.node.common.node_table.syft_object import SYFT_OBJECT_VERSION_1
+from .....core.node.common.node_table.syft_object import SyftObject
+from .....oblv.constants import DOMAIN_CONNECTION_PORT
+from .....oblv.constants import LOCAL_MODE
+from .....oblv.constants import WORKER_MODE
+from .....oblv.deployment_client import OblvMetadata
+from ....common.serde.deserialize import _deserialize as deserialize
+from ....common.serde.serializable import serializable
+from ....common.uid import UID
+from ...common.exceptions import OblvEnclaveError
+from ...common.exceptions import OblvProxyConnectPCRError
+from ..api import SyftAPI
+from ..api import UserNodeView
+from ..client import HTTPConnection
+from ..client import Routes
+from ..context import AuthedServiceContext
+from ..context import ChangeContext
+from ..credentials import SyftSigningKey
+from ..document_store import DocumentStore
+from ..service import AbstractService
+from ..service import service_method
+from ..user_code import UserCode
+from ..user_code import UserCodeStatus
+from .oblv_keys import OblvKeys
+from .oblv_keys_stash import OblvKeysStash
+from .util import find_available_port
+
+# caches the connection to Enclave using the deployment ID
+OBLV_PROCESS_CACHE: Dict[str, List] = {}
+
+
+# TODO: 🟡 Duplication of PyPrimitive Dict
+# This is emulated since the action store curently accepts  only SyftObject types
+@serializable(recursive_serde=True)
+class DictObject(SyftObject):
```

Comment:
Can this just be an AnyActionObject?

Reply:
meaning subclassing from AnyActionObject?